topics/metabolomics/tutorials/gcms/tutorial.md

```diff
@@ -1,37 +1,70 @@
 ---
 layout: tutorial_hands_on
-draft: true
+draft: false
 
 title: 'Mass spectrometry : GC-MS analysis with metaMS package'
 level: Introductory
-zenodo_link : 'https://zenodo.org/record/3244991' 
-questions : 
-    - What are the main steps of GC-MS datas processing for metabolomic analysis ? 
-    - How te be able to annotate the maximum of unknowns using Galaxy ? 
-objectives : 
-    - To be sure you have already comprehend the diversity of MS pre-processing analysis. 
-    - To learn the principal functions of metaMS package through Galaxy.
-    - To evaluate the potential of this new GC-MS workflow for GC-MS metabolomic analysis. 
-time_estimation : 2H 
-key_points : 
-    - Have a good file containing all your peaks during the first stopover 
-    - Find all your unknowns in your datas
-    - Find your stanards if you have some 
-requirements :
-  - type: "internal"
-    topic_name: metabolomics
-    tutorials: 
-      - lcms
-contributors : 
+zenodo_link: 'https://zenodo.org/records/16538501' 
+questions: 
+- What are the main steps for gas chromatography-mass spectrometry (GC-MS) data processing for untargeted metabolomic analysis?
+- How to conduct metabolomic GC-MS data analysis from preprocessing to annotation using Galaxy?
+objectives: 
+- To be sure you have already comprehend the diversity of MS pre-processing analysis. 
+- To learn the principal functions of metaMS package for GC-MS data processing through Galaxy.
+- To evaluate the potential of two workflow approaches and available Galaxy tools when dealing with GC-MS metabolomic analysis. 
+time_estimation: 2H 
+key_points: 
+- To process untargeted GC-MS metabolomic data preprocessing, you need a large variety of steps and tools.
+- Although main steps are standard, various ways to combine and to set parameters for tools exist, depending on your data.
+- Resources are available in Galaxy, but do not forget that you need appropriate knowledge to perform a relevant analysis. 
+requirements:
+- type: "internal"
+  topic_name: metabolomics
+  tutorials:
+    - gcms
+contributions:
+  autorship: 
     - jsaintvanne
-
+    - yguitton
+  editing:
+    - yguitton
+    - melpetera
+    - jsaintvanne
+  testing:
+    - workflow4metabolomics
+funding:
+    - elixir-europe
+    
 --- 
 
+You may already know that there are different types of *-omic* sciences; out of these, metabolomics is most closely related to phenotypes. Metabolomics involves the study of different types of matrices, such as blood, urine, tissues, in various organisms including plants. It  focuses on studying the very small molecules which are called *metabolites*, to better understand matters linked to the metabolism. However, studying metabolites is not a piece of cake since it requires several critical steps which still have some major bottlenecks. Metabolomics is still quite a young science, and has many kinds of specific challenges.
+{: .text-justify}
```

Comment:
Styling like this should not be set on the tutorial level, to keep the look and feel consistent throughout tutorials. 

We can consider updating the GTN-wide template to always justify text, but should then do some extensive testing e.g. that it works for all tutorials and different size screens etc.